app/templates/main/record.html

```diff
@@ -59,49 +59,111 @@
+                            <div class="govuk-grid-column-two-thirds">
+                                <div class="details-container">
+                                    <h1 class="govuk-heading-m govuk-heading-m__record-details">Record details</h1>
+                                    <dl class="govuk-summary-list govuk-summary-list--record">
+                                        {% with dict = list_items %}
+                                            {% include "summary_list_items.html" %}
+                                        {% endwith %}
+                                    </dl>
+                                </div>
+                            </div>
+                            <div class="govuk-grid-column-one-third govuk-grid-column-one-third--record mobile-record-layout-rights">
+                                <div class="rights-container">
+                                    <h3 class="govuk-heading-m govuk-heading-m__rights-header">Rights to access</h3>
+                                    <a href="{{ url_for('main.download_record', record_id=record['file_id']) }}"
+                                       class="govuk-button govuk-button__download--record"
+                                       data-module="govuk-button">Download
+                                    record</a>
+                                    {% if download_filename %}
+                                        <p class="govuk-body govuk-body--download-filename">
+                                            The downloaded record will be named
+                                            <br>
+                                            <strong>{{ download_filename }}</strong>
+                                        </p>
+                                    {% endif %}
+                                    <p class="govuk-body govuk-body--terms-of-use">
+                                        Refer to <a href="/terms-of-use" class="govuk-link govuk-link--ayr">Terms of use.</a>
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="govuk-grid-column-one-third govuk-grid-column-one-third--record mobile-record-layout-record-arrangement">
+                                <div class="record-container">
+                                    <h2 class="govuk-heading-m govuk-heading-m__record-header">Record arrangement</h2>
+                                    <ol class="record-arrangement-list">
+                                        {% for file_level in record["file_path"].split("/") %}
+                                            <li class="govuk-body__record-arrangement-list">{{ file_level }}</li>
+                                        {% endfor %}
+                                    </ol>
+                                </div>
+                            </div>
+                        </div>
+                        <!-- Record HTML View -->
```

Comment:
as discussed, we're going to park html view for now, so this block can be removed

Reply:
Removed any references to Record HTML view